indico/web/forms/fields.py

```diff
@@ -830,6 +830,8 @@ def __init__(self, *args, **kwargs):
         self.protected_object = kwargs.pop('protected_object')(kwargs['_form'])
         get_acl_message_url = kwargs.pop('acl_message_url', None)
         self.acl_message_url = get_acl_message_url(kwargs['_form']) if get_acl_message_url else None
+        self.iacl_count = len([x for x in self.protected_object.protection_parent.get_access_list(skip_managers=True)
+                               if x not in [entry.principal for entry in self.protected_object.acl_entries]])
```

Comment:
actually it's the same code.. maybe it'd make sense to add a `get_inherited_...` like method to ProtectionMixin?